packages/next/src/root-layout/index.tsx

```diff
@@ -30,3 +32,132 @@ export async function RootLayout( {
 		</html>
 	);
 }
+
+type IconTypes = {
+	faviconIcons: Icon[];
+	appleIcons: Icon[];
+	msApplicationTileIcon: Icon;
+};
+
+type Icon = {
+	sourceUrl: string;
+	height: string;
+	width: string;
+};
+
+type FormattedIcon = {
+	url: string;
+	sizes: string;
+};
```

Comment:
Move types and interfaces to `@snapwp/types`

Reply:
Related (possible disagreement - didnt look into how the types are used, naming convention there was enough for me to 🚩 ): https://github.com/rtCamp/snapwp/pull/119#discussion_r1986430854